src/front/glsl/functions.rs

```diff
@@ -236,38 +251,42 @@ impl Program<'_> {
                             body,
                         )))
                     }
+                    "atan" => match args.len() {
+                        1 => Ok(Some(ctx.add_expression(
```

Comment:
nit: the match could return `Expression`, which we'd then add outside of the match